core/src/test/java/io/questdb/griffin/ExpressionParserTest.java

```diff
@@ -215,6 +215,22 @@ public void testNotInReverseContext() throws Exception {
         x("ax'a''b'innotand", "a and not x in ('a','b')");
     }
 
+    @Test
+    public void testIsNull() throws SqlException {
```

Comment:
indeed, thank you.

Reply:
I've checked Postgres and it seems to support `<literal> is [not] null`. Probably, we should to the same, so that `3 is null`, `null is null`, `'abc' is not null`, and `$1 is not null` are all allowed. WDYT?

Reply:
Do you mean supporting this:

- table where 3 is null -> to return 0 rows as `3 is null` evaluates to false?
- table where null is null -> to return all rows as `null is null` evaluates to true?
- table where 'abc' is not null -> to return all rows as `'abc' is not null` evaluates to true?
- table where $1 is not null -> ? is this to select the name of the column dynamically? ?? is that allowed?

Reply:
Items 1-3 - yes. As for the item 4, `$1` here is a bind variable.